packages/ui/components/BasicInfoForm/BasicInfoForm.tsx

```diff
@@ -1,8 +1,7 @@
 import React, { useRef } from 'react';
-import { InputField } from 'components/InputField';
 import styles from 'styles/workspace.module.css';
-import { Button } from 'components/Button';
-
+import formstyles from './basicform.module.css';
+import { Button, Input } from 'antd';
```

Comment:
Can we use Ant Steps component https://ant.design/components/steps/

Reply:
sure

Reply:
@brettstack can you please review?